blocks/section/__intro/_place/section__intro_place_form.css

```diff
@@ -0,0 +1,16 @@
+.section__intro_place_form {
+  max-width: 756px; 
+  margin: 100px 0 84px;
+}
+@media screen and (max-width: 1199px) {
+  .section__intro_place_form {
+    max-width: 490px;
+    margin: 100px 0 82px;
+  }
+}
+@media screen and (max-width: 659px) {
+  .section__intro_place_form {
+    max-width: 422px;
```

Comment:
I've checked the layout and the paragraph has a width of 280px on the mobile screen. I think you should change it.

Reply:
@MarPostovik done